### DISCUSSION TITLE
RuneLite Memory Leak (Apple Silicon Client), >10 GB of Memory after ~4 Hours

### DISCUSSION THREAD

**Discussion by Choodz** [runelite/runelite#16429] [Q&A]
The memory use increases with time. This was confirmed by checking Activity Monitor periodically. 

Platform Information:
- MacBook Pro M2 Max
- macOS Ventura 13.2.1
- RuneLite version 2.6.3 (for Apple Silicon)

Active Plugins (that may affect memory use):
- 117 HD
- Animation Smoothing

I have yet to test without the above plugins active, as I love 117 HD and never want to play without it again. 

But something is clearly wrong. If I left this open for long enough, it would surely eat through all my RAM and eventually crash.

![Screenshot 2023-03-27 at 4 38 46 PM](https://user-images.githubusercontent.com/80929256/228075980-c136fc51-5150-481b-9270-b7b22a92c9df.png)

**Comment by Nightfirecat** [comment]
This is very likely to be a graphics driver memory leak. I recommend playing for a while with safe mode on (see below) and GPU plugin disabled and seeing whether this is reproduceable.

To open in safe mode on Windows: In Command Prompt, run:
```
"%localappdata%\runelite\runelite.exe" --safe-mode
```

Or on Mac: In Terminal, run:
```
/Applications/RuneLite.app/Contents/MacOS/RuneLite --safe-mode
```

**Reply by Choodz** [reply]
Although it breaks my heart to play without HD, I will follow your recommendation and report back with the results.

**Comment by Choodz** [comment]
Ran in Safe Mode (which de-activates GPU plugins by default) for over four hours and there was no significant increase in memory use.

I have instead opened an issue ticket on the 117 HD GitHub [here](https://github.com/117HD/RLHD/issues/184).
